components/InventoryItemListItemKebab/index.tsx

```diff
@@ -43,6 +30,38 @@ export default function InventoryItemListItemKebab({
   const handleCloseKebabMenu = () => {
     setAnchorElKebab(null)
   }
+  const router = useRouter()
+  const option: InventoryItemListItemKebabOption[] = [
+    {
+      name: 'Check in',
+      onClick: () =>
+        router.push(
+          `/checkIn?inventoryItem=${encodeURIComponent(
+            JSON.stringify(inventoryItem)
+          )}`
+        ),
+    },
+    {
+      name: 'Check out',
+      onClick: () =>
+        router.push(
+          `/checkOut?inventoryItem=${encodeURIComponent(
+            JSON.stringify(inventoryItem)
+          )}`
+        ),
+    },
+    {
+      name: 'Delete',
+      onClick: () => {
+        if(window.confirm("Are you sure you want to delete this from the inventory?")) {
```

Comment:
We should probably use a modal for this instead of browser alerts. (Though I guess this would be another story...)

Reply:
is there a reason not to use browser alerts?